addons/binding/org.openhab.binding.amazondashbutton/README.md

```diff
@@ -9,51 +9,47 @@ The binding uses pcap4j (https://www.pcap4j.org/) in order to capture ARP and BO
 
 ## Install Native Library
 
-Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J.
+Let’s start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer. They are native libraries that powers the core functionalities of Pcap4J. Pcap4J needs administrator/root privileges.
 
-### Ubuntu
+Source: [https://www.pcap4j.org/](https://www.pcap4j.org/)
+
+### Debian/Ubuntu
 
 ```
 apt-get install libpcap-dev
 ```
 
-### CentOs
+Note: You can run Pcap4J with a non-root user by granting capabilities `CAP_NET_RAW` and `CAP_NET_ADMIN`
+to your java command by the following command: 
 
 ```
-yum install libpcap-devel
+sudo setcap cap_net_raw,cap_net_admin=eip `realpath /usr/bin/java`
 ```
 
-### Mac
+Be aware of other capabilities which are set by setcap. **These capabilities will be overwritten!** If you need more capabilities (like "cap_net_bind_service" for the Network binding), you have to add them like this :
```

Comment:
Looks good. "Be aware of previously set capabilities"? Maybe mention getcap (see Network binding) to give the user the ability to actually check him/herself?